src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();
+        Color[] colors = Color.values();
+        return colors[random.nextInt(colors.length)].toString();
```

Comment:
in enum it is better to use the name() method
```suggestion
        return colors[random.nextInt(colors.length)].name();
```

Reply:
i did change method to method  that proposal you